src/components/navigation/weekday-selection/WeekdaySelection.tsx

```diff
@@ -99,8 +108,93 @@ const WeekdaySelection = () => {
     [selectedDate, selectedDateVarWrapper],
   );
 
+  const [isDragging, setIsDragging] = useState(false);
+  const [xCoord, setXCoord] = useState<number | null>(null);
```

Comment:
```suggestion
  const [xCoordinate, setXCoordinate] = useState<number | null>(null);
```
If you use `dragDirection` instead of `dragDir` below, I think it is more consistent to not abbreviate such terms in the first place.

Reply:
Agreed, changed in
```
refactor(weekday-selection): rename `xCoord` property
```

Reply:
LGTM!